app/Service/Student/StudentBootcampDetailService.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Service\Student;
+
+use App\Models\Bootcamp;
+use App\Models\Student;
+
+class StudentBootcampDetailService
+/**
```

Comment:
You can remove all the comment as the function is already typed and we can easily know what are the parameters and what is the function returning. Leave the throws as is the only relevant data that is worth to keep. 

In general terms, always try to reduce documentation to the minimum so the maintenance of the documentation becomes reduced too. In this case, as this is a service, the class name should be able to describe what this service is doing.

Reply:
Ahhh, ok! Now I really understood it.